model/DataStore/MetaDataDeliverySyncTask.php

```diff
@@ -44,30 +44,48 @@ class MetaDataDeliverySyncTask extends AbstractAction implements JsonSerializabl
 {
     use OntologyAwareTrait;
 
+    public const INCLUDE_DELIVERY_METADATA_PARAM_NAME = 'includeMetadata';
```

Comment:
chore: does it make sense to rename the class to match it's new purpose as it is now processing not only deliveries but also tests, what do you think?

Reply:
In my opinion better would be to have separate classes and reuse common logic. What if we would introduce couple of more entities? How that would look like?

Reply:
Separate task has been created for test processing